crates/typst/src/model/content.rs

```diff
@@ -61,109 +64,181 @@ use crate::util::pretty_array_like;
 /// elements the content is composed of and what fields they have.
 /// Alternatively, you can inspect the output of the [`repr`]($repr) function.
 #[ty(scope)]
-#[derive(Debug, Clone, Hash)]
-#[allow(clippy::derived_hash_with_manual_eq)]
-pub struct Content {
-    elem: Element,
-    attrs: EcoVec<Attr>,
-}
-
-/// Attributes that can be attached to content.
-#[derive(Debug, Clone, PartialEq, Hash)]
-enum Attr {
-    Span(Span),
-    Field(EcoString),
-    Value(Prehashed<Value>),
-    Child(Prehashed<Content>),
-    Styles(Styles),
-    Prepared,
-    Guard(Guard),
-    Location(Location),
-}
+#[derive(Debug, Clone)]
+pub struct Content(Arc<dyn NativeElement>, Element);
```

Comment:
Why is the `Element` now stored here? Isn't it trivial to get via the trait?

Reply:
Well you left a comment about removing the dynamic dispatch by casting to a `dyn Any` which unfortunately is not possible (it would require a dynamic dispatch), however what *is* possible is storing the `Element` since it doesn't increase the size of `Value`.